manubot/cite/citeproc.py

```diff
@@ -1,16 +1,17 @@
+"""Correct or validate CSL item schema.
+
+Module naming: citeproc is the generic name for programs that produce 
+formatted bibliographies and citations based on the metadata of 
+the cited objects and the formatting instructions provided by 
+Citation Style Language (CSL) styles. 
+-- https://en.wikipedia.org/wiki/CiteProc
+"""
 import copy
 import functools
 import logging
 import re
 
-csl_item_type_fixer = {
-    'journal-article': 'article-journal',
-    'book-chapter': 'chapter',
-    'posted-content': 'manuscript',
-    'proceedings-article': 'paper-conference',
-    'standard': 'entry',
-    'reference-entry': 'entry',
-}
+from manubot.cite.csl_item import CSL_Item
 
 
 def csl_item_passthrough(csl_item, set_id=None, prune=True):
```

Comment:
If we go with the deepcopy, let's updated this docstring to include something like, "Return a new CSL_Item..."

Reply:
The argument should be CSL_Item type on input and output, and yes, returning a new value is safer.